src/reachy2_sdk/reachy_sdk.py

```diff
@@ -615,19 +615,19 @@ def is_off(self) -> bool:
 
     def set_pose(
         self,
-        common_pose: str = "zero",
+        common_pose: str = "default",
```

Comment:
There are references to the zero poses in the examples, notebook and tests to be changed also